README.md

```diff
@@ -2,7 +2,7 @@
 The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
 
-![screenshot](https://raw.githubusercontent.com/microverseinc/curriculum-javascript/main/leaderboard/images/leaderboard_wireframe.png?token=AQE6HSPX3NJHSIM2YAMRRKLBBE7V6)
+![screenshot](leaderboard.png)
 
 ## Built With
 
```

Comment:
- [ ] You should also add instructions for using API. (where and how to generate any necessary keys, credentials, how to set, store, and generate them) 

Consider someone who is not from the microverse may want to check your project.
If I want to use your app for myself, currently there is no way to know how to configure (API) it.

Reply:
doen 👍